pkg/controllers/management/etcdbackup/etcdbackup.go

```diff
@@ -250,14 +244,17 @@ func (c *Controller) rotateExpiredBackups(cluster *v3.Cluster, clusterBackups []
 	internvalHours := cluster.Spec.RancherKubernetesEngineConfig.Services.Etcd.BackupConfig.IntervalHours
 	expiredBackups := getExpiredBackups(retention, internvalHours, clusterBackups)
 	for _, backup := range expiredBackups {
+		if backup.Spec.Manual {
```

Comment:
can you confirm that this code will result in manually taking snapshots not being removed, for both local and and the ones backed up to S3

Reply:
Ya, all backup configs have `manual` irrespective of provider flag, https://github.com/rancher/rancher/blob/6ff16315e29ff7a1ace33413a19a4232617d7493/pkg/controllers/management/etcdbackup/etcdbackup.go#L285